content/en/docs/concepts/configuration/even-pods-spread.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Even Pods Spread
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+This page explains how EvenPodsSpread feature works and the mechanics under the hood.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## EvenPodsSpread
+
+`EvenPodsSpread` feature gives users more fine-grained control on the pods' distribution across the cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
```

Comment:
Some of this belongs in the page overview. When revising, consider these tweaks:
```suggestion
To use _spread constraints_, you need to enable the `EvenPodsSpread` https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
```

Reply:
We can simply remove this section as the content has been moved to the "overview" section.